content/en/user/run-your-own.md

```diff
@@ -23,18 +25,18 @@ Here is what you need:
 
 - A **domain name**. This is how you and others will access your server and how you and your users will be identified on the network.
 
-  **How to get**: Namecheap, Gandi, any of the infinite number of domain name registrars. Comes with a yearly cost that varies depending on domain name choice.
-- A **VPS**. Something that will run the Mastodon code that will always be connected to the internet.
+  **How to get**: Namecheap, Gandi, Hover, or any of the large number of domain name registrars. A domain comes with an annual cost, that varies depending on the choice of domain name.
+- A **VPS** (Virtual Private Server). Something that will run the Mastodon code, that will always be connected to the internet.
 
-  **How to get**: DigitalOcean, Hetzner, Exoscale, Scaleway, any of the infinite number of hosting providers. Comes with a monthly or yearly cost that varies depending on hardware specifications.
+  **How to get**: DigitalOcean, Hetzner, Exoscale, Scaleway, any of the large number of hosting providers. A VPS comes with a monthly or annual cost that varies depending on hardware specifications.
 - An **e-mail provider**. Mastodon needs to send confirmation links and various notifications through e-mail, and hosting your own SMTP server, while possible, is much more difficult to do reliably than to use a third-party provider.
 
-  **How to get**: Mailgun, SparkPost, Postmark, Sendgrid, any of the infinite number of e-mail hosting providers that expose a SMTP API. Comes with a monthly cost based on volume of e-mails sent.
-- Optional: **Object storage provider**. Mastodon can save files that you and your users upload on the hard disk drive of the VPS it runs on, however, the hard disk drive is usually not infinite and difficult to upgrade later. An object storage provider gives you practically infinite metered file storage.
+  **How to get**: Mailgun, SparkPost, Postmark, Sendgrid, any of the large number of e-mail hosting providers that expose a SMTP API. An email provider comes with a monthly or annual cost, usually based on volume of e-mails sent.
```

Comment:
`any of the large number of e-mail hosting providers` -> `or any email hosting provider`

`expose a SMTP API` -> `expose an SMTP API`

also maybe worth trying to reconcile usage of `email` vs `e-mail` throughout the page? (no personal preference on which one we use, except perhaps a weak preference for `email` as this appears less dated and more modern)